tsc/OpenTimelineIO_TSC_Charter.md

```diff
@@ -0,0 +1,67 @@
+  * **c.** TSC projects generally will involve Contributors and Committers. The TSC may adopt or modify roles so long as the roles are documented in the CONTRIBUTING file. Unless otherwise documented:
+    * **i.** Contributors include anyone in the technical community that contributes code, documentation, or other technical artifacts to the Project;
+    * **ii.** Committers are Contributors who have earned the ability to modify ("commit") source code, documentation or other technical artifacts in a project's repository; and
+    * **iii.** A Contributor may become a Committer by a majority approval of the existing Committers. A Committer may be removed by a majority approval of the other existing Committers.
+  * **d.** Participation in the Project through becoming a Contributor and Committer is open to anyone so long as they abide by the terms of this Charter.
+  * **e.** The TSC may (1) establish work flow procedures for the submission, approval, and closure/archiving of projects, (2) set requirements for the promotion of Contributors to Committer status, as applicable, and (3) amend, adjust, refine and/or eliminate the roles of Contributors, and Committers, and create new roles, and publicly document any TSC roles, as it sees fit.
+  * **f.** The TSC may elect a TSC Chair, who will preside over meetings of the TSC and will serve until their resignation or replacement by the TSC. The TSC Chair, or any other TSC member so designated by the TSC, will serve as the primary communication contact between the Project and the Technical Advisory Council of the Academy Software Foundation of The Linux Foundation.
+  * **g.** Responsibilities: The TSC will be responsible for all aspects of oversight relating to the Project, which may include:
+    * **i.** coordinating the technical direction of the Project;
+    * **ii.** approving project or system proposals (including, but not limited to, incubation, deprecation, and changes to a sub-project's scope);
+    * **iii.** organizing sub-projects and removing projects;
+    * **iv.** creating sub-committees or working groups to focus on cross-project technical issues and requirements;
+    * **v.** appointing representatives to work with other open source or open standards communities;
+    * **vi.** establishing community norms, workflows, issuing releases, and security issue reporting policies;
+    * **vii.** approving and implementing policies and processes for contributing (to be published in the CONTRIBUTING file) and coordinating with the Series Manager to resolve matters or concerns that may arise as set forth in Section 7 of this Charter;
```

Comment:
"Series Manager" is not defined in this document, I am vague as to who that is, I think it should be stated. Their role however is seemingly explicit in the several mentions in this document.

Reply:
Ah, I thought I recalled that this had been raised before. I feel the rest of the issues mentioned in #6, related to LF definitions and entities have been resolved in this current charter, with the exception of the Series Manager. In the context of this charter's current text, I don't know what a "Series" is, but I do know that a "Series Manager" helps coordinate resolving things. Since the series manager coordinates, as opposed to approving, blocking, or otherwise gating the activity of the TSC, the lack of clarity around the role does not seem to impede the business of the TSC. @jmertic, @jminor, does that sound right? If I've understood it properly, I'm happy to mark this conversation as "resolved, fine as is".

Reply:
Correct @meshula - the Series Manager is more of a legal role and does not impact day-to-day operations

Reply:
Thanks!